src/main/resources/en/planet.yml

```diff
@@ -0,0 +1,97 @@
+en:
+  faker:
+    planet:
+      name:
+      - "Jupiter"
+      - "Mars"
+      - "Venus"
+      - "Uranus"
+      - "Mercury"
+      - "Neptune"
+      - "Earth"
+      - "Saturn"
+      length_of_day:
```

Comment:
What is the source of all this data in the file that we could double check and keep as a refernce?

Reply:
agree, requires more data. I'll add more info later on. Let's keep it as draft for now.

In general info gathered from lots of sources.

I'll add more from Wikipedia then :)